src/gmp/ConnectionManager.h

```diff
@@ -79,6 +82,10 @@ class ConnectionManager : public ExceptionListener {
 
 	void registerHandler(pGiapiErrorHandler handler);
 
+        Connection* getConnection();
```

Comment:
why are we exposing the connection? It seems the purpose of the `ConnectionManager` is to hide those details

Nitpick: the code seems misaligned